hadoop/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
 [#347]: https://github.com/stackabletech/docker-images/pull/347
 [#400]: https://github.com/stackabletech/docker-images/pull/400
 
+### Changed
+
+- Upgraded to JMX Exporter 0.19 ([#400])
```

Comment:
Please move it into existing `Changed` section and change number to 431

Reply:
Uhm...this whole thing was accidental. I'll check why this is in here.

Reply:
This is actually also correct. This is the Hadoop changelog...but you're correct in that the Airflow changelog hasn't been updated.